python/python/glide/async_commands/core.py

```diff
@@ -205,37 +209,71 @@ class TrimByMinId(StreamTrimOptions):
     Stream trim option to trim by minimum ID.
     """
 
-    def __init__(self, exact: bool, threshold: str, limit: Optional[int] = None):
+    def __init__(self, threshold: str):
+        """
+        Initialize trim option by minimum ID.
+
+        Args:
+            threshold (str): Threshold for trimming by minimum ID.
+        """
+        super().__init__(threshold, "MINID", None, None)
+
+    def __init__(self, exact: bool, threshold: str):
         """
         Initialize trim option by minimum ID.
 
         Args:
             exact (bool): If `true`, the stream will be trimmed exactly.
                 Otherwise the stream will be trimmed in a near-exact manner, which is more efficient.
             threshold (str): Threshold for trimming by minimum ID.
-            limit (Optional[int]): Max number of entries to be trimmed. Defaults to None.
-                Note: If `exact` is set to `True`, `limit` cannot be specified.
         """
-        super().__init__(exact, threshold, "MINID", limit)
+        super().__init__(threshold, "MINID", exact, None)
+
+    def __init__(self, threshold: str, limit: int):
+        """
+        Initialize trim option by minimum ID.
+
+        Args:
+            threshold (str): Threshold for trimming by minimum ID.
+            limit (int): Max number of entries to be trimmed. Defaults to None.
```

Comment:
There is another constructor without limit, maybe skip that sentence? Also, limit is not optional, so user can't pass None.
```suggestion
            limit (int): Max number of entries to be trimmed.
```